### PR TITLE
Upgrade Docker images to latest available

### PR DESCRIPTION
**Pull request by risdenk**
3 commits broken down into changing `docker-compose.yml`, `rre/Dockerfile`, and `blacklight/Dockerfile`

### CODE REVIEW

File: rre/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM maven:3.6.3-adoptopenjdk-11
+FROM maven:3-eclipse-temurin-11
```
Comment (epugh) [inline rre/Dockerfile:1]: Can you provide context on why this one versust he other one?  Just curious!

**Comment by risdenk** [comment]
`adoptopenjdk` became `eclipse-temurin` - https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/

The other subtle change here is moving from specific Maven version 3.6.3 - to the latest 3.x maven version. 3.8.4 is the current Maven 3.x version. This gives up some reproducibility for making sure latest Maven 3.x docker image is used. There is a `3.8.4-eclipse-temurin-11` tag if would prefer locking the Maven 3.x version.

File: rre/Dockerfile (hunk 1 above)
Comment (risdenk) [reply rre/Dockerfile:1]: `adoptopenjdk` became `eclipse-temurin` - https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/

The other subtle change here is moving from specific Maven version 3.6.3 - to the latest 3.x maven version. 3.8.4 is the current Maven 3.x version. This gives up some reproducibility for making sure latest Maven 3.x docker image is used. There is a `3.8.4-eclipse-temurin-11` tag if would prefer locking the Maven 3.x version.

Comment (epugh) [reply rre/Dockerfile:1]: awesome!

**Pull request merged**